src/main/java/Nehsik.java

```diff
@@ -0,0 +1,93 @@
+import java.util.Scanner;
+public class Nehsik {
+    public static final int MAX_TASKS = 100;
+
+    public static void main(String[] args) {
+        displayGreetings();
+        
+        Scanner in = new Scanner(System.in);
```

Comment:
Consider closing the scanner with in.close() at the end of your program to follow best practices and avoid resource leaks